documentation/modules/ROOT/pages/configuration/topic-auto-create-config.adoc

```diff
@@ -13,19 +13,69 @@
 
 toc::[]
 
-{prodname} automatically creates *internal* topics for offsets, connector status, config
-storage and history topics. The destination topics for the captured tables will be
-automatically created with a default config by the Kafka brokers when
-`auto.create.topics.enable` is set to `true`.{empty} +
-When topic creation is disabled on the brokers, for example in production environments,
-or when the topics need a different configuration then these topics have to be created
-upfront either automated in a custom deployment process or manually until Kafka Connect 2.6.
+Beginning with Kafka 2.6.0, you can set the Kafka Connect configuration to permit automatic creation of topics.
+If you enable automatic topic creation,
+when Debezium emits a change event record, Kafka Connect ingests the record into Kafka and writes it to a topic.
```

Comment:
This part of the sentence is always true, also without the condition on the line before.